client/my-sites/patterns/patterns.tsx

```diff
@@ -1,18 +1,16 @@
+import { useTranslate } from 'i18n-calypso';
 import Main from 'calypso/components/main';
 
 import './style.scss';
 
 export default () => {
+	const translate = useTranslate();
+
 	return (
-		<Main fullWidthLayout isLoggedOut>
-			<div>
-				<h1>Build your perfect site with patterns</h1>
+		<Main isLoggedOut fullWidthLayout>
+			<h1>Build your perfect site with patterns</h1>
 
-				<p>
-					Hundreds of expertly designed, fully responsive patterns allow you to craft a beautiful
-					site in minutes.
-				</p>
-			</div>
+			<p>{ translate( 'Trending' ) }</p>
```

Comment:
I've borrowed that string from the `Tags` [page](https://wordpress.com/tags) to be able to use one that has all the translations available. I will remove it in a future iteration.